src/main/groovy/org/boozallen/plugins/jte/init/primitives/JteNamespace.groovy

```diff
@@ -0,0 +1,53 @@
+package org.boozallen.plugins.jte.init.primitives
+
+import hudson.AbortException
+
+import javax.annotation.Nonnull
+
+class JteNamespace extends TemplatePrimitive implements Serializable{
+
+    private static final long serialVersionUID = 1L
+
+    List<Namespace> namespaces = []
```

Comment:
I think namespaces should be a Map and push should use the namespace.getName() as the key